File: python/compare_the_triplets.py
#!/bin/python3

def compare_triplets(points_alice, points_bob):
    score_alice = 0
    score_bob = 0
    i = 0
    while i < len(points_alice):
        try:
            alice = int(points_alice[i])
            bob = int(points_bob[i])
            if alice > bob:
                score_alice = score_alice + 1 
            elif alice < bob:
                score_bob = score_bob + 1
            i = i + 1
        except Exception as ex:
            print(ex)
            break
    print(score_alice, score_bob)

points_alice = input().strip().split(" ")
points_bob = input().strip().split(" ")
if len(points_alice) == len(points_bob):
    compare_triplets(points_alice, points_bob)

File: java/CompareTheTriplets.java
import java.util.Arrays;
import java.util.Scanner;

public class CompareTheTriplets {

    public static void main(String... args) {
        Scanner in = new Scanner(System.in);
        String[] pointsAlice = in.nextLine().split(" ");
        String[] pointsBob = in.nextLine().split(" ");
        if (pointsAlice.length == pointsBob.length) {
            compareTriplets(pointsAlice, pointsBob);
        }
    }

    private static void compareTriplets(String[] pointsAlice, String[] pointsBob) {
        int i = 0;
        int scoreAlice = 0;
        int scoreBob = 0;
        while (i < pointsAlice.length) {
            try {
                final int alice = Integer.valueOf(pointsAlice[i]);
                final int bob = Integer.valueOf(pointsBob[i]);
                if (alice > bob) {
                    scoreAlice = scoreAlice + 1;
                } else if (alice < bob) {
                    scoreBob = scoreBob + 1;
                }
                i = i + 1;
            } catch (Exception ex) {
                System.err.println(ex.getMessage());
                break;
            }
        }
        System.out.println(scoreAlice + " " + scoreBob);
    }

}